### ISSUE TITLE
pom.xml:188-194: In the continuation of #588, all the...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/cactoos#1166]
The puzzle `1119-fd48e847` from #1119 has to be resolved: 

https://github.com/yegor256/cactoos/blob/a17252e2a30697dbd2de97aaff662aad5fc0ade6/pom.xml#L188-L194

The puzzle was created by @rultor on 20-Jul-19. 

Estimate: 30 minutes,  role: DEV. 

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@dddff4c1-9156-49e2-a2d9-cf978c5a9655, thread: pool-506-thread-1
com.zerocracy.farm.sync.SyncProject[76] java.lang.IllegalStateException: Failed to acquire "roles.xml" in "C63314D6Z" in 2min: java.util.concurrent.locks.ReentrantLock@6d09edc9[Locked by thread pool-470-thread-1]
```

[0.48.2](https://github.com/zerocracy/farm): Issue: yegor256/cactoos#1166, Action: `opened`

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@dddff4c1-9156-49e2-a2d9-cf978c5a9655, thread: pool-11-thread-6
com.zerocracy.farm.sync.SyncProject[76] java.lang.IllegalStateException: Failed to acquire "wbs.xml" in "C63314D6Z" in 2min: java.util.concurrent.locks.ReentrantLock@6d09edc9[Locked by thread pool-470-thread-1]
```

[0.48.2](https://github.com/zerocracy/farm): CID: [0478fdc7-647c-4400-9814-962b97a94e71](https://www.0crat.com/footprint/C63314D6Z/0478fdc7-647c-4400-9814-962b97a94e71), Type: "Add job to WBS"

**Comment by victornoel** [comment]
@paulodamaso I think this should be put manually in scope because of a ([known](https://github.com/zerocracy/farm/issues/2119)) 0crat bug

**Comment by paulodamaso** [comment]
@0crat in

**Comment by 0crat** [comment]
> @0crat in ([here](https://github.com/yegor256/cactoos/issues/1166#issuecomment-534141389))

@paulodamaso Job [#1166](https://github.com/yegor256/cactoos/issues/1166) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#1166](https://github.com/yegor256/cactoos/issues/1166) assigned to @victornoel[/z](https://www.0crat.com/u/victornoel), here is [why](https://www.0crat.com/footprint/C63314D6Z/4dc2a3d0-0b3d-4843-8923-654cc6be071a); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by victornoel** [comment]
@0crat wait for #1197 to be merged

**Comment by 0crat** [comment]
> @0crat wait for #1197 to be merged ([here](https://github.com/yegor256/cactoos/issues/1166#issuecomment-536203957))

@victornoel The impediment for [#1166](https://github.com/yegor256/cactoos/issues/1166) was registered successfully by @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by victornoel** [comment]
@paulodamaso I'm not sure why, 0pdd didn't react to the change to the todo, could you close this issue manually? thanks

**Comment by paulodamaso** [comment]
@victornoel Looks like the puzzle hasn't been changed at all, would you please make another PR with the fix?

**Comment by 0pdd** [comment]
The puzzle `1119-fd48e847` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd 2 puzzles [#1222](https://github.com/yegor256/cactoos/issues/1222), [#1223](https://github.com/yegor256/cactoos/issues/1223) are still not solved.

**Comment by 0crat** [comment]
The job [#1166](https://github.com/yegor256/cactoos/issues/1166) is now out of scope

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @victornoel[/z](https://www.0crat.com/u/victornoel), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Quality review completed: +4 point(s) just awarded to @sereshqua[/z](https://www.0crat.com/u/sereshqua)

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @victornoel[/z](https://www.0crat.com/u/victornoel)

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1223](https://github.com/yegor256/cactoos/issues/1223) is still not solved; solved: [#1222](https://github.com/yegor256/cactoos/issues/1222).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1434](https://github.com/yegor256/cactoos/issues/1434) is still not solved; solved: [#1222](https://github.com/yegor256/cactoos/issues/1222), [#1223](https://github.com/yegor256/cactoos/issues/1223).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1583](https://github.com/yegor256/cactoos/issues/1583) is still not solved; solved: [#1222](https://github.com/yegor256/cactoos/issues/1222), [#1223](https://github.com/yegor256/cactoos/issues/1223), [#1434](https://github.com/yegor256/cactoos/issues/1434).

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1910](https://github.com/yegor256/cactoos/issues/1910) is still not solved; solved: [#1222](https://github.com/yegor256/cactoos/issues/1222), [#1223](https://github.com/yegor256/cactoos/issues/1223), [#1434](https://github.com/yegor256/cactoos/issues/1434), [#1583](https://github.com/yegor256/cactoos/issues/1583).
